packages/squidex/src/entities/team.ts

```diff
@@ -14,13 +13,10 @@ export interface Team<T = string> {
     description?: string;
     name: string;
   }[];
-  outputs: T[];
 }
 
 export interface RestTeam extends Entity, Rest<Team> {}
 // TODO: REMOVE old proposal type
-export interface GraphqlTeam
-  extends Entity,
-    Graphql<Team<GraphqlResearchOutput | { id: string }>> {
+export interface GraphqlTeam extends Entity, Graphql<Team<{ id: string }>> {
```

Comment:
is this still being used anywhere? I think we should have moved to the generated queries by now